### ISSUE TITLE
SFTP: Password setting not working as expected

### ISSUE THREAD

**Issue by SuperSandro2000** [rclone/rclone#3672]
#### What is the problem you are having with rclone?

When setting up an SFTP remote you get this:
````
SSH password, leave blank to use ssh-agent.
y) Yes type in my own password
g) Generate random password
n) No leave this optional password blank
y/g/n>
y/g/n>
y/g/n>
````
Pressing enter does repeat the prompt instead of using `n` which is also does not suggest it uses SSH Agent then.

#### What is your rclone version (output from `rclone version`)

rclone v1.49.5
- os/arch: linux/amd64
- go version: go1.12.10

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)

Linux Ubuntu

####  Which cloud storage system are you using? (eg Google Drive)

SFTP

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

rclone config

**Comment by ncw** [comment]
> Pressing enter does repeat the prompt instead of using `n` which is also does not suggest it uses SSH Agent then.

Pressing `n` is the right thing to do here to use the ssh agent.

What do you think rclone should do?

**Comment by SuperSandro2000** [comment]
The thing it suggested to do:
``leave blank to use ssh-agent.``
Use the ssh-agent when I enter nothing.

**Comment by ncw** [comment]
The help text is supposed to work like this

> SSH password, **leave blank** to use ssh-agent.
> y) Yes type in my own password
> g) Generate random password
> n) No **leave** this optional password **blank**

Unfortunately the text "SSH password..." is also used in the options

      --sftp-pass string            SSH password, leave blank to use ssh-agent.

I could make the `y/g/n` prompt make `n` be the same as Return.

**Comment by SuperSandro2000** [comment]
That sounds way more logical. I support those changes.

**Comment by ncw** [comment]
I've fixed this now :-)

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.51
